parkhangs-webapp/src/components/EventDetailModal.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import './DetailModal.css'
+import { connect } from 'react-redux'
+
+
+class EventDetailModal extends React.Component {
+    constructor(props) {
+        super(props)
+        this.state = {
```

Comment:
Hardcoded Placeholder to show one specific park for demo purposes
TODO: filter according to specific event in specified park.